src/commands/environmentVariables/editEnvironmentVariable/EnvironmentVariableEditContext.ts

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { type EnvironmentVar } from "@azure/arm-appcontainers";
+import { type SetTelemetryProps } from "../../../telemetry/SetTelemetryProps";
+import { type ContainerUpdateTelemetryProps as TelemetryProps } from "../../../telemetry/commandTelemetryProps";
+import { type ISecretContext } from "../../secret/ISecretContext";
+import { type EnvironmentVariablesBaseContext } from "../EnvironmentVariablesContext";
+import { type EnvironmentVariableType } from "../addEnvironmentVariable/EnvironmentVariableTypeListStep";
+
+export interface EnvironmentVariableEditBaseContext extends EnvironmentVariablesBaseContext, Pick<ISecretContext, 'secretName'> {
```

Comment:
Seems to be very similar to the `EnvironmentVariableAddBaseContext`. Any reason to make a new one?

Reply:
Yeah you're right, they are super similar.  To simplify a bit, I extended from the add context and just made the `environmentVariable` required because it's needed up front for the edits.  I think I still prefer to keep a separate context just because I'm trying to keep the two command paths decoupled for the most part.